request/consumers/base.py

```diff
@@ -13,26 +13,23 @@ def __init__(self, key):
         self.app_base_route = ''
 
     def make_request(self, method, query=None, body=None):
-        try:
-            connection = HTTPSConnection(self.host, context=ssl._create_unverified_context())
-        except AttributeError:
-            connection = HTTPSConnection(self.host)
-        params = ''
+        payload = ''
         if query is not None:
-            params = '?{}'.format(urlencode(query))
-        path = '{}/{}/{}/{}'.format(
-            self.app_base_route,
-            self.key,
-            method,
-            params,
-        )
-        connection.request('GET', path, body=body)
-        response = connection.getresponse()
-        data = response.read()
+            payload = query
+        try:
+            r = requests.get('https://{}/{}/{}/{}'.format(
+                self.host,
+                self.app_base_route,
+                self.key,
+                method,
+            ), params=payload)
+        except:
+            sys.exc_info()[0]
```

Comment:
Would prefer an explicit log and return None here:

    except Exception:
        print(
            sys.exc_info()[0],
            file=sys.stderr,
        )
        return None